Simplified/Book/UI/NYPLBookDetailView+Accessibility.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  NYPLBookDetailView+Accessibility.swift
+//  Simplified
+//
+//  Created by Ernest Fan on 2021-07-21.
+//  Copyright © 2021 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+
+@objc extension NYPLBookDetailView {
+  @objc(setUIAccessibilityFocusToView:)
+  func setUIAccessibilityFocus(to view: UIView) {
+    if UIAccessibility.isVoiceOverRunning {
```

Comment:
I would put the `isVoiceOverRunning` check inside the main thread closure, just to be safe. Also in the other method below.